src/main/java/de/zalando/ep/zalenium/container/ContainerClient.java

```diff
@@ -5,7 +5,7 @@
 import java.util.Map;
 
 public interface ContainerClient {
-    
```

Comment:
Please revert changes to this file, it is just some spaces and tabs that changed.